doc/user/content/sql/prepare.md

```diff
@@ -0,0 +1,59 @@
+---
+title: "PREPARE"
+description: "`PREPARE` creates a prepared statement (a server-side object that can be used to optimize performance)."
+menu:
+  main:
+    parent: "sql"
+---
+
+`PREPARE` creates a prepared statement. A prepared statement is a server-side object that can be used to optimize performance by separating planning from data.
+
+`PREPARE` parses, analyzes, and rewrites the initial `SELECT`, `INSERT`, `UPDATE`, or`DELETE` statement. A subsequent [`EXECUTE`] statement then plans and executes the statement. You can use also use `PREPARE` in conjunction with [`EXPLAIN`] to gain insight into a query's performance impact.
+
+## Syntax
+
+{{< diagram "prepare.svg" >}}
+
+Field | Use
+------|-----
+name | A name for this particular prepared statement that you can later use to execute or deallocate a statement. The name must be unique within a session.
+datatype  |  The data type of a parameter for the prepared statement.
+statement  |  Any `SELECT` statement.
+
+## Details
+
+Prepared statements can take parameters: values that are substituted into the statement when it is executed. If the data type of a particular parameter is unspecified or is specified as unknown, it will be inferred from the context in which the parameter is first referenced. To refer to the parameters in the prepared statement itself, use `$1`, `$2`, etc.
```

Comment:
We don't support specifying param types, so the second sentence should have its first half removed.